cluster/examples/kubernetes/ceph/filesystem-ec.yaml

```diff
@@ -17,6 +17,8 @@ spec:
       size: 3
   # The list of data pool specs
   dataPools:
+    - replicated:
```

Comment:
And let's add the comment here. Better to document in multiple places so it's more discoverable